docker-compose.yml

```
@@ -79,7 +86,8 @@ services:
   st2workflowengine:
     image: ${ST2_IMAGE_REPO:-stackstorm/}st2workflowengine:${ST2_VERSION:-3.3.0}
     restart: on-failure
-    depends_on: ["st2api"]
+    depends_on:
+      - st2api
```

I'm pretty sure that WorkflowEngine also relies on the coordination backend: 

```suggestion
    depends_on:
      - redis
      - st2api
```